tang-operator/Sanity/runtest.sh

```diff
@@ -1018,6 +1019,58 @@ rlJournalStart
     rlPhaseEnd
     ############# /LEGACY TESTS ###########
 
+    ############# DAST TESTS ############
+    ### Only execute DAST TESTS if helm command exists ...
+    type helm 2>/dev/null 1>/dev/null && {
```

Comment:
I did not know this construction but the following is probably the most correct one
```suggestion
    command -v helm >/dev/null && {
```